signac/contrib/project.py

```diff
@@ -353,15 +354,19 @@ def config(self):
         """
         return self._config
 
+    @deprecated(
+        deprecated_in="1.8",
+        removed_in="2.0",
+        current_version=__version__,
+        details="Use Project.path instead.",
+    )
     def root_directory(self):
-        """Return the project's root directory.
+        """Alias for :meth:`~Project.path`."""
```

Comment:
Double-check if `:meth:` or `:attr:` is the correct way to refer to properties. I can't remember.

Reply:
It should be `:attr:`, I just copy-pasted and forgot to change it. Thanks.